scripts/reportportal/rp_manual_reporter/collector.py

```
@@ -0,0 +1,476 @@
+    """
+    fixtures: list[str] = []
+
+    for decorator in decorators:
+        if not isinstance(decorator, ast.Call):
+            continue
+
+        is_mark, marker_name = _is_pytest_mark_attr(node=decorator.func)
+        if is_mark and marker_name == "usefixtures":
+            for arg in decorator.args:
+                if isinstance(arg, ast.Constant) and isinstance(arg.value, str):
+                    fixtures.append(arg.value)
```

Ruff (0.15.18)

[warning] Use `list.extend` to create a transformed list

(PERF401)